tests/jerry/regexp-construct.js

```diff
@@ -98,3 +98,28 @@ r3 = new RegExp(foo)
 
 assert (r1.source === r2.source);
 assert (r2.source === r3.source);
+
+/* RegExp properties */
+assert (RegExp.length === 2);
+assert (RegExp.prototype.source === "");
```

Comment:
`print ((new RegExp('')).source);` outputs '(?:)'.

Shouldn't the value be the same?